kaldi/install.bash

```diff
@@ -5,6 +5,7 @@
 # Clone kaldi fork from tue-robotics (only latest commit)
 KALDI_REPO="https://github.com/tue-robotics/kaldi.git"
 KALDI_HOME=~/src/kaldi_speech
+KALDI_REPO_BRANCH="techunited"
```

Comment:
Why are we  not working on our own master branch?

Reply:
Would make rebasing on the original repo more confusing, this is fine

Reply:
Because if we end up finding a bug in kaldi, we will not be able to submit a PR as the guidelines of kaldi don't allow PRs of forked master. 

I am also going to revert the master branch of our fork back to the original one.

It will also keep our changes/implementations separate from the master.